samples/dashboards/demo/climate/test-notes.md

```diff
@@ -1 +1 @@
-Click on the world map
+Click on a city marker in the world map
```

Comment:
Also, click on the KPI?

Reply:
Good idea! And switching the temperature scale.